src/pages/edge/edge-network/tabs/index.md

```diff
@@ -28,116 +34,71 @@ implementation 'com.adobe.marketing.mobile:edgeidentity'
 
 <Alerts query="platform=android-gradle&componentClass=InlineNestedAlert"/>
 
-2. Import the Mobile Core and Edge extensions in your application class.
-
-```java
-import com.adobe.marketing.mobile.MobileCore;
-import com.adobe.marketing.mobile.Edge;
-```
-
-<Variant platform="ios" task="add" repeat="7"/>
+<Variant platform="ios-pods" task="add" repeat="2"/>
 
-1. Add the Mobile Core and Edge extensions to your project using CocoaPods. Add following pods in your `Podfile`:
+Add the required dependencies to your project using CocoaPods. Add following pods in your `Podfile`:
 
 ```swift
 use_frameworks!
+
 target 'YourTargetApp' do
-pod 'AEPCore', '~> 5.0'
-pod 'AEPEdge', '~> 5.0'
-pod 'AEPEdgeIdentity', '~> 5.0'
+  pod 'AEPCore', '~> 5.0'
+  pod 'AEPEdge', '~> 5.0'
+  pod 'AEPEdgeIdentity', '~> 5.0'
 end
 ```
 
-2. Import the Mobile Core and Edge libraries:
-
-#### Swift
-
-```swift
-// AppDelegate.swift
-import AEPCore
-import AEPEdge
-import AEPEdgeIdentity
-```
-
-#### Objective-C
-
-```objectivec
-// AppDelegate.h
-@import AEPCore;
-@import AEPEdge;
-@import AEPEdgeIdentity;
-```
-
-<Variant platform="android" task="register" repeat="4"/>
-
-#### Java
-
-```java
-public class MainApp extends Application {
-
-  private final String ENVIRONMENT_FILE_ID = "YOUR_APP_ENVIRONMENT_ID";
-
- @Override
- public void onCreate() {
-  super.onCreate();
-
-  MobileCore.setApplication(this);
-  MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
-
-  MobileCore.registerExtensions(
-   Arrays.asList(Edge.EXTENSION, Identity.EXTENSION),
-   o -> Log.d("MainApp", "Adobe Experience Platform Mobile SDK was initialized.")
-  );
- }
-}
-```
+<Variant platform="android-kotlin" task="add-simplified-initialization" repeat="1"/>
```

Comment:
You can remove all the initialization code from this page as you are importing it from `resources/initilaize.md`.